goes16-l2-cmi.json

```diff
@@ -0,0 +1,569 @@
+    "temporal_composition_schema": null,
+    "composition_function": null,
+    "collection_type": "collection",
+    "metadata": {
+        "providers": [
+            {
+                "url": "https://www.nasa.gov/content/goes",
+                "name": "National Aeronautics and Space Administration (NASA)",
+                "roles": [
+                    "producer"
+                ]
+            },
+            {
+                "url": "https://www.goes-r.gov/",
+                "name": "National Oceanic and Atmospheric Administration (NOAA)",
+                "roles": [
+                    "processor",
+                    "producer",
+                    "licensor"
+                ]
+            },
+            {
+                "url": "https://data.inpe.br/big/",
+                "name": "National Institute for Space Research (INPE)",
+                "roles": [
+                    "host",
+                    "processor"
+                ]
+            }
+        ],
+        "license": {
+            "type": "Public Domain",
+            "uri": "https://www.nesdisia.noaa.gov/policy.html"
+        }
+    },
+    "keywords": [
+        "goes",
+        "goes-16",
+        "level-2",
+        "cloud",
+        "moisture",
+        "meteorology",
+        "atmosphere",
+        "surface",
+        "brazil"
+    ],
+    "is_public": true,
+    "is_available": true,
+    "category": "eo",
+    "version": 1,
+    "version_predecessor": null,
+    "version_successor": null,
+    "extent": {
+        "spatial": {
+            "bbox": [
+            [
+              -180,
+              -81.33,
+              6.3,
+              81.33
+            ],
+            [
+              -116.0,
+              -56.0,
+              -25.0,
+              35.0
+            ]
+          ]
+        }
+    },
+    "summaries": {
```

Comment:
We are adopting the pattern of platform, constellation and instruments using lower case

Reply:
Ok!